statics/js/components/topology.js

```diff
@@ -613,6 +625,11 @@ var TopologyComponent = {
         self.addFilterK8sTypes(highlight, "storage", ["persistentvolume", "persistentvolumeclaim", "storageclass"]);
       }
 
+      if (self.isIstioEnabled()) {
+	self.addFilterIstioTypes(filter, "network", ["cluster", "container", "namespace", "pod", "virtualservice", "gateway"]);
```

Comment:
these two lines need to be indented properly, a space is missing

Reply:
DONE